packages/devtools_app/lib/src/standalone_ui/vs_code/devtools_extensions.dart

```diff
@@ -232,6 +244,14 @@ extension on EditorDebugSession {
     // Since all paths are absolute, assume that if the path contains `\` and
     // not `/` then it's Windows.
     final isWindows = rootPath.contains(r'\') && !rootPath.contains(r'/');
+
+    // Ensure the returned URI ends with a trailing slash to match the format of
+    // the project root URIs returned by DTD.
+    final endsWithSlash =
+        isWindows ? rootPath.endsWith(r'\') : rootPath.endsWith('/');
+    if (!endsWithSlash) {
+      rootPath = '$rootPath${isWindows ? r'\' : '/'}';
+    }
```

Comment:
nit: this might be simpler and have less wrapping with a variable for the slash

```
final slash = isWindows ? r'\' : '/';
if (!rootPath.endsWith(slash)) rootPath += slash;
```